.github/workflows/test.yaml

```diff
@@ -12,6 +12,9 @@ on:
   push:
 
 jobs:
```

Comment:
Can we make all the tests run on master every time for a while until we are confident in this?

Reply:
done